packages/cli/src/commands/setup/auth/templates/auth.js.template

```diff
@@ -17,7 +17,17 @@ export const getCurrentUser = async (
   { _token, _type },
   { _event, _context }
 ) => {
-  return { ...decoded, roles: parseJWT({ decoded }).roles }
+  if (decoded) {
```

Comment:
This makes sense, but I would just put the return first 
```js
if(!decoded) {
 return null
}
```

but just personal preference here